src/common/database.c

```diff
@@ -2498,36 +2498,20 @@ void dt_database_show_error(const dt_database_t *db)
 
     char *label_text = g_markup_printf_escaped(
         _("\n"
-          " At startup, the database failed to open because at least one of the two files in the database is locked.\n"
-          "\n"
-          " The persistence of the lock is mainly caused by one of the two following causes:\n"
-          "\n"
-          " - Another occurrence of darktable has already opened this database file and locked it for its benefit.\n"
-          "\n"
-          " - A previous occurrence of darktable ended abnormally and therefore \n"
-          "   could not close one or both files in the database properly.\n"
+          " Sorry, darktable could not be started (database is locked)\n"
           "\n"
           " How to solve this problem?\n"
           "\n"
-          " 1 - Search in your environment if another darktable occurrence is active. If so, use it or close it. \n"
-          "     The lock indicates that the process number of this occurrence is : <i><b>%d</b></i>\n"
-          "\n"
-          " 2 - If you can't find this other occurrence, try closing your session and reopening it or shutting down your computer. \n"
-          "     This will delete all running programs and thus close the database correctly.\n"
-          "\n"
-          " 3 - If these two actions are not enough, it is because at least one of the two files that materialize the locks remains \n"
-          "     and that these are no longer attached to any occurrence of darktable. It is then necessary to delete it (or them). \n"
-          "     The two files are named <i>data.db.lock</i> and <i>library.db.lock</i> respectively. The opening mechanism signals \n"
-          "     the presence of the <i><b>%s</b></i> file in the <i><b>%s</b></i> folder. \n"
-          "     (full pathname: <i><b>%s</b></i>).\n"
+          " 1 - check if darktable is already running (might be process id %d)\n"
           "\n"
-          "     <u>Caution!</u> Do not delete these files without first checking that there are no more occurrences of darktable, \n"
-          "     otherwise you risk generating serious inconsistencies in your database.\n"
+          " 2 - if you can't find it -> restart your computer\n"
```

Comment:
To short for me. Sometimes (in Linux especially) just closing/reopen session is ok instead of completely restart. I suggest to precise just that: "close/reopen your session or restart your computer"